app/web/features/auth/locales/de.json

```diff
@@ -7,12 +7,13 @@
     }
   },
   "account_form": {
-    "tos_accept_label": "Ich akzeptiere die $t(global:terms_of_service).",
+    "tos_accept_label": "Ich akzeptiere die Nutzungsbedingungen.",
```

Comment:
I think it would be better to update the `global:terms_of_service` variable translation rather than removing it.